cfn-resources/search-index/CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Changelog
+
+## (2023-10-12)
```

Comment:
Is this a breaking change? I think so?

Reply:
Yes this would be breaking change. We could analyse deprecating (and not removing) the existing "Fields" property, and add a new one which expects the new json format.